code/datums/mind.dm

```diff
@@ -284,7 +284,7 @@
 
 /datum/mind/proc/has_antag_datum(datum_type, check_subtypes = TRUE)
 	if(!datum_type)
-		return
+		return TRUE
```

Comment:
What's the logic behind this change?
This proc returned either a reference or `null`. Adding a boolean to the mix is not good for type safety.